rpc/params/logstream.go

```diff
@@ -7,24 +7,27 @@ import (
 	"time"
 )
 
-// LogStreamRecord contains a slice of LogStreamRecords.
+// TODO(debug-log) - these structs are used to stream logs to the rsyslog forward worker
+// Delete when the worker is removed.
+
+// LogStreamRecords contains a slice of LogStreamRecords.
 type LogStreamRecords struct {
 	Records []LogStreamRecord `json:"records"`
 }
 
 // LogStreamRecord describes a single log record being streamed from
 // the server.
 type LogStreamRecord struct {
-	ID        int64     `json:"id"`
-	ModelUUID string    `json:"mid"`
-	Entity    string    `json:"ent"`
-	Version   string    `json:"ver,omitempty"`
-	Timestamp time.Time `json:"ts"`
-	Module    string    `json:"mod"`
-	Location  string    `json:"lo"`
-	Level     string    `json:"lv"`
-	Message   string    `json:"msg"`
-	Labels    []string  `json:"lab"`
+	ID           int64             `json:"id"`
+	ModelUUID    string            `json:"mid"`
+	Entity       string            `json:"ent"`
+	Timestamp    time.Time         `json:"ts"`
+	Module       string            `json:"mod"`
+	Location     string            `json:"lo"`
+	Level        string            `json:"lv"`
+	Message      string            `json:"msg"`
+	LegacyLabels []string          `json:"lab,omitempty"`
+	Labels       map[string]string `json:"lbl,omitempty"`
```

Comment:
that's one way

Reply:
ah, meant to not use LegacyLabels for now. will delete that. another PR will do the compatibility stuff